OracleHTTPServer/README.md

```diff
@@ -41,7 +41,7 @@ If we want to start the OHS container with some pre-specified mod_weblogic confi
 
          For e.g
          $ cd /scratch/DockerVolume/OHSVolume
-         $ docker run -v `pwd`:/volume -w /volume -d --name ohs -p 7777:7777  oracle/ohs:12.2.1.2.0-sa configureWLSProxyPlugin.sh
+         $ docker run -v `pwd`:/u01/oracle/ohssa/user_projects -w /u01/oracle/ohssa/user_projects -d --name ohs -p 7777:7777  oracle/ohs:12.2.1.2.0-sa configureWLSProxyPlugin.sh
```

Comment:
I'd caution against using `pwd` here. Instead of `cd`'ing into `/scratch/DockerVolume/OHSVolume` first, just specify it directly so that the user's current directory is not important:

```
$ docker run -v /scratch/DockerVolume/OHSVolume:/u01/oracle/ohssa/user_projects -w /u01/oracle/ohssa/user_projects -d --name ohs -p 7777:7777  oracle/ohs:12.2.1.2.0-sa
```